lib/interface_quda.cpp

```diff
@@ -4932,6 +4934,7 @@ void updateGaugeFieldQuda(void* gauge,
    if (param->use_resident_gauge) {
      if (!gaugePrecise) errorQuda("No resident gauge field to use");
      cudaGauge = gaugePrecise;
+     gaugePrecise = nullptr;
```

Comment:
Is this addition safe? I thought `gaugePrecise` could incidentally get used by other routines. Whether or not that happens in practice though MILC is another story.

Reply:
removed.